source/2020-01-27-the-ember-times-issue-132.md

```diff
@@ -56,13 +55,33 @@ If you have a great idea for a meetup, you can [sign up here](https://emberconf.
 
 ---
 
-## [Section title in sentence case 🐹](#section-url)
+## [Try your first Octane & JAM sandwich at EmberConf 🏎🥪🍓](https://emberconf.com/#/speakers/chris-manson)
 
-<change section title emoji>
-<consider adding some bold to your paragraph>
+Surely, you remember him as [the Emberista who built a blog site with Ember & NodeJs on stage in less than 15 mins](https://www.youtube.com/watch?v=GHbjdesHhPA), you heard him talk about [his work on an Ember-powered blog engine](https://www.youtube.com/watch?v=OnDUp2AhuNo) or you
+recall his [jammiest introduction to the JAM (JavaScript, APIs and Markup) stack for humans](https://www.youtube.com/watch?v=eQULGb10MbQ) with [46 mentions of the word "JAM" in total](https://twitter.com/real_ate/status/1207683401446313984)!
```

Comment:
I think we can switch the order to add more humor:
```suggestion
recall him [say JAM 46 times](https://twitter.com/real_ate/status/1207683401446313984) in his [jammiest introduction to the JAM (JavaScript, APIs and Markup) stack for humans](https://www.youtube.com/watch?v=eQULGb10MbQ)!
```

Reply:
Thank you, that sounds great!